src/main/java/com/gluonhq/substrate/model/ReleaseConfiguration.java

```diff
@@ -258,6 +271,22 @@ public void setAppLabel(String appLabel) {
         this.appLabel = appLabel;
     }
 
+    public String getAppDescription() {
+        return appDescription == null ? "" : appDescription;
```

Comment:
Why do we need an empty string, probably null is good enough as default?

Reply:
If we return null from the configuration, we need to add a null check in MSIBundler instead. It should be the responsibility of the Data class to return the correct data.

Reply:
That would apply to the rest of the fields of this class then. But this is the only case. And I don't see a problem doing null checks?

Reply:
There isn't an issue with null checks. However, there are some methods like `getBundleVersion` which returns null and then we check for the value and assign a default value which is fetched from parameter placed in the same class `ReleaseConfiguration#DEFAULT_BUNDLE_VERSION`.

I feel that ReleaseConfiguration methods can take care of these null checks and return default values, if required. But, as you said, it will require change to the entire class. I will update the code as you suggested for now.